test_shared/build.gradle

```diff
@@ -26,33 +26,33 @@ android {
 dependencies {
     api project(':clevertap-core')
     // Unit ing dependencies
-    api Libs.junit_platform_runner
-    api Libs.junit_jupiter_api
-    api Libs.junit_jupiter_engine
-    api Libs.kotlin_stdlib_jdk7
-    api Libs.jsonassert
-    api Libs.gson
+    api (libs.test.junit.platform.runner)
+    api (libs.test.junit.jupiter.api)
+    api (libs.test.junit.jupiter.engine)
+    api (libs.kotlin.stdlib.jdk7)
+    api (libs.test.jsonassert)
+    api (libs.gson)
     def mockito_version = '3.5.11'
```

Comment:
@william-ct check if we can move this version catalog as well?

Reply:
Done